tools/cli/adminElasticSearchCommands.go

```diff
@@ -260,6 +261,7 @@ func AdminDelete(c *cli.Context) {
 }
 
 func parseIndexerMessage(fileName string) (messages []*indexer.Message, err error) {
+	// #nosec
```

Comment:
@vancexu please review and provide comment.

Reply:
correct, no security concern here since file read as pure text.